pontoon/base/static/js/translate.js

```diff
@@ -1953,6 +1964,27 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Update authors list in filter menu
+     */
+    updateAuthors: function () {
+      $('#filter').find('.authors, .for-authors').toggle(this.authors.length > 0);
+      var authors = $('#filter .authors ul').empty();
+
+      $.each(this.authors, function() {
+        authors.append('<li data-type="' + this.email + '">' +
+          '<figure class="author">' +
+            '<img class="rounded" src="' + this.gravatar_url + '">' +
+            '<figcaption>' +
+              '<p class="name">' + this.display_name + '</p>' +
+              '<p class="email">' + this.email + '</p>' +
```

Comment:
I know It's a bigger topic but maybe we should change .email into username to protect contributors privacy? At some point of course.